src/common/marketUtils.es6

```diff
@@ -11,6 +11,26 @@ const getMarketPosition = (() => {
     };
 })();
 
+const getMarketChild = (markets) => {
+    return markets.map((market) => {
+        if (market.submarkets) {
+            const submarkets_sort = market.submarkets.sort((a, b) => {
+                if (a.display_name < b.display_name) return -1;
+                if (a.display_name > b.display_name) return 1;
+            });
+            market.submarkets = submarkets_sort;
```

Comment:
I think this will mutate the market passed in as a argument, to decouple and avoid side-effects better to:

`const getMarketChild = ([...markets]) => {`